src/modules/user/application/dtos/create-user.dto.ts

```diff
@@ -0,0 +1,5 @@
+export class CreateUserDto {
+  readonly nickName: string;
+  readonly email: string;
+  readonly password: string;
+}
```

Comment:
You can use validations in your dtos: https://docs.nestjs.com/techniques/validation.